src/ocm/components/clusterConfiguration/networkConfiguration/SubnetsDropdown.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { DropdownItem, DropdownToggle, Dropdown } from '@patternfly/react-core';
+import { CaretDownIcon } from '@patternfly/react-icons';
+import { useField } from 'formik';
+import { getFieldId } from '../../../../common';
+
+type SubnetsDropdownProps = {
+  name: string;
+  items: {
```

Comment:
`items` is very generic which become confusing in the future. Let's maybe use a more specific name e.g. `machineSubnets`?